### DISCUSSION TITLE
can I build eGRN with base cistromes

### DISCUSSION THREAD

**Discussion by wangmhan** [aertslab/scenicplus#125] [Q&A]
Hello,

Thank you for developing the nice SCENIC+ package!

I wonder if it is possible to run eGRN analysis only on scRNA-seq data? I have base cistromes from bulk data: TF-to-gene relationship, so that would like to skip the preprocess. 

It will be great if there is an demo dataset, so that can understand the scplus_obj better, and easier to fix if my own project run into trouble due to the wrong format.

Thank you in advance!

**Comment by SeppeDeWinter** [comment]
Hi @wangmhan 

Thank you for your kind words.

I transferred your question from the issues to the discussions.

The problem here would be inferring region-to-gene links, in order to be able to do this you will need matched scRNA-seq and scATAC-seq data. If you can somehow get this information from a different source it should in theory be possible to run SCENIC+ with your data.

As for an example/demo dataset I would suggest to have a look at our tutorials: https://scenicplus.readthedocs.io/en/latest/tutorials.html .

Best,

Seppe
